cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/KubeTwillRunnerService.java

```diff
@@ -76,48 +75,59 @@
  * and annotations:
  *
  * cdap.twill.app=[literal app name]
+ *
+ * @param <T> type of the K8s resource
```

Comment:
I don't think this is needed right? We only need one `KubeTwillRunnerService` for all different resource types?

Reply:
Yes. We will have single `KubeTwillRunnerService` for all different resources types created by single pod. That is appfabric and preview pod will have different instances of the `KubeTwillRunnerService` , however any kube resources created by these pods will use same instance. Currently appfabric only creates `DEPLOYMENTS` and preview only creates `STATEFULSETS` though.